cmd/buildah/commit.go

```diff
@@ -20,6 +21,10 @@ var (
 			Name:  "signature-policy",
 			Usage: "`pathname` of signature policy file (not usually used)",
 		},
+		cli.BoolFlag{
+			Name:  "quiet",
```

Comment:
Fixed.  I'd shy away from making --quiet a global option if in a number of cases it would have no effect.